config/finders/all_content.yml

```diff
@@ -6,16 +6,12 @@ locale: en
 publishing_app: rummager
 rendering_app: finder-frontend
 schema_name: finder
-title: All content
 description: Find content from government
 signup_content_id: 49838f97-5d2e-407a-8876-30c4abe2a6bc
 details:
   default_documents_per_page: 20
   document_noun: result
   format_name: Documents
-  reject:
-    content_purpose_supergroup:
-    - other
```

Comment:
😢 can we remove `reject_content_purpose_supergroup` from the email alert api if we're not using this anymore?

Reply:
yes good idea

Reply:
Added a trello card for this